authorization-dotnet/BouncyCastleCryptography/AsymmetricEncryption/DsaEncryptor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Org.BouncyCastle.Crypto.Generators;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Security;
+using System.Text;
+
+namespace BouncyCastleCryptography.AsymmetricEncryption
```

Comment:
We are trying to follow file-scoped namespace declaration of all the projects now. Here also we can start following the same.

This comment is application throughout the project